src/components/Table/ImageTable.js

```diff
@@ -170,9 +170,10 @@ ImageRow.propTypes = {
 const ImageTable = (props) => {
   const intl = useIntl();
   // start sorting by created date
-  const [activeSortIndex, setActiveSortIndex] = useState(null);
-  const [isSortAscending, setIsSortAscending] = useState(true);
-  const [sortBy, setSortBy] = useState("id");
+  const [activeSortIndex, setActiveSortIndex] = useState("job_created");
```

Comment:
So if you set this to `useState(3)` it will set the sort index as expected. The defaultSort code isn't necessary imo.

Reply:
Without the check of the `defaultSort` the Created column remains as selected even after selecting another sort index
![Screenshot from 2023-02-01 14-18-08](https://user-images.githubusercontent.com/49452678/216057818-0f525934-d070-4a9b-a2e4-fe51a81cf4f7.png)

Renamed to `initialSort` as that name might be a better fit.